src/frontend/hooks/server/invites.ts

```diff
@@ -30,25 +30,17 @@ export function useAcceptInvite(projectId?: string) {
 
   return useMutation({
     mutationFn: async ({inviteId}: {inviteId: string}) => {
-      if (!inviteId) return;
       return mapeoApi.invite.accept({inviteId});
     },
-    onSuccess: () => {
-      // This is a workaround. There is a race condition where the project in not available when the invite is accepted. This is temporary and is currently being worked on.
-      setTimeout(() => {
-        Promise.all([
-          queryClient.invalidateQueries({
-            queryKey: [INVITE_KEY],
-          }),
-          queryClient.invalidateQueries({
-            queryKey: [ALL_PROJECTS_KEY],
-          }),
-        ]).then(() => {
-          if (projectId) {
-            switchActiveProject(projectId);
-          }
-        });
-      }, 5000);
```

Comment:
the workaround described here does not seem to be relevant anymore. did a basic test with the replacement implementation and it seemed to behave as expected. the only thing that _may_ need to be adjusted is potentially invalidating the caches before switching projects e.g.

```ts
// Invalidate the queries first
await Promise.all([...])

// Then switch projects
switchActiveProject(...)
```